test/e2e/cluster.sh

```diff
@@ -240,7 +240,8 @@ if [ "${UPGRADE_CLUSTER}" = "true" ]; then
       --auth-method client_secret \
       --client-id ${AZURE_CLIENT_ID} \
       --client-secret ${AZURE_CLIENT_SECRET} || exit 1
-
+    # wait a few mins for added/deleted vms to sync with the k8s api node registration state
+    sleep 5m
```

Comment:
Five minutes? That's like...forever.

Reply:
I know, and it didn't even work!

Alternate approach incoming...